kpi/models/asset_file.py

```diff
@@ -158,12 +159,15 @@ def get_download_url(self, request):
 
     @staticmethod
     def get_path(asset, file_type, filename):
+        default_storage = get_storage_class()()
+        valid_filename = default_storage.get_valid_name(filename)
```

Comment:
can `PRIVATE_STORAGE_CLASS` be something different from what's returned by this function?

Reply:
idea:
```
In [15]: AssetFile._meta.get_field('content').storage
Out[15]: <private_storage.storage.files.PrivateFileSystemStorage at 0x7f8d053781c0>

In [16]: AssetFile._meta.get_field('content').storage.get_valid_name('ridic/\n\\ulous')
Out[16]: 'ridiculous'
```